src/test/java/com/exasol/sql/ddl/drop/TestDropSchema.java

```diff
@@ -24,13 +24,13 @@ void getTableName() {
 
     @Test
     void getIfExistsFalseByDefault() {
-        assertFalse(this.dropSchema.getIfExists());
+        assertFalse(this.dropSchema.hasIfExistsModifier());
```

Comment:
I don't get what this method is doing after I read the name :(

Reply:
Returns true if the statement has "IfExists",